### ISSUE TITLE
Can't stop the music...

### ISSUE THREAD

**Issue by simatao** [MattVador/node-red-contrib-play-sound#3]
Hi Matt,
the sound node plays mp3s fine, but I cannot stop the music. I use a inject node with msg.payload = "stop". It successfully removes the "playing" icon underneath the sound node and triggers the "stop" response on the sound node's output. So I guess the node understands the stop command, because it behaves differently when I send gibberish inputs. But, the music keeps playing nontheless.
I am using a raspberry Pi 3, all updated & upgraded and hdmi sound output. Node Version 6.11.1. Node Red version 0.17.4
Please help me fix this.
Thanks

**Comment by simatao** [comment]
Hi, the issue seems to occur when using HDMI for audio output. When I managed to use the audio-jack I could control it. Great Node otherwise! :-D

**Comment by sanderloonen** [comment]
Hello MattVador,

I am afraid that this is still an issue.
There is no response to the "stop" payload (nor "pause") on the Mini-Jack
on a Raspberry pi3 running Node-red 0.19.6, node.js 10.15.1

Here is a sample Node-red patch.

I hope that you can fix this otherwise excellent node.
thanks

`[{"id":"1226b862.a3c8a8","type":"PlaySound","z":"a992881b.240248","name":"hello","playerOptions":"{}","audioURI":"/home/pi/Music/Hello.wav","options":"{}","x":562,"y":284,"wires":[["ff1b253a.d96ec8"]]},{"id":"ff1b253a.d96ec8","type":"debug","z":"a992881b.240248","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"payload","x":903.5,"y":294,"wires":[]},{"id":"56121db3.5b1d04","type":"inject","z":"a992881b.240248","name":"","topic":"","payload":"play","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":174.5,"y":245,"wires":[["1226b862.a3c8a8"]]},{"id":"e2fbba98.02b158","type":"inject","z":"a992881b.240248","name":"","topic":"","payload":"stop","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":174.5,"y":311,"wires":[["1226b862.a3c8a8"]]}]
`

**Comment by MattVador** [comment]
Hi, have you tried to run package 'play-sound' in nodejs (without nodered) ?

**Comment by sanderloonen** [comment]
HI MattVador, 

I have not. I dont really have the skills at hand to do this, so it would take me a while to learn. 
Unfortunatly, I made it work with turning the volume to 0 and making the flow wait untill the sound file is finished. (I found a work around)
have to move on to other projects...

I will keep an eye on this though, it will come up again.....


thanks for all your help.

Ciao

Sander
